.github/workflows/label-conflicting-pr.yml

```
@@ -0,0 +1,20 @@
+name: "Label conflicting PRs"
+on:
+  push:
+  pull_request_target:
+    types: [synchronize]
+    
+permissions:
+  pull-requests: write
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Label conflicting PRs
+        uses: eps1lon/actions-label-merge-conflict@v2
```

what I meant was this:
```suggestion
        uses: eps1lon/actions-label-merge-conflict@b8bf8341285ec9a4567d4318ba474fee998a6919 # v2.0.1
```

(not a blocker from my side though)

---

oic 👀

---

Applying this change would be nice. (Can Dependabot handle this?)

---

It's supposed to be able to https://github.com/dependabot/dependabot-core/issues/2470

although there's at least one report of it not working right